server/etcdmain/config.go

```diff
@@ -189,8 +189,7 @@ func newConfig() *config {
 	fs.StringVar(&cfg.ec.DNSClusterServiceName, "discovery-srv-name", cfg.ec.DNSClusterServiceName, "Service name to query when using DNS discovery.")
 	fs.StringVar(&cfg.ec.InitialCluster, "initial-cluster", cfg.ec.InitialCluster, "Initial cluster configuration for bootstrapping.")
 	fs.StringVar(&cfg.ec.InitialClusterToken, "initial-cluster-token", cfg.ec.InitialClusterToken, "Initial cluster token for the etcd cluster during bootstrap.")
-	fs.Var(cfg.cf.clusterState, "initial-cluster-state", "Initial cluster state ('new' or 'existing').")
-
+	fs.Var(cfg.cf.clusterState, "initial-cluster-state", "Initial cluster state ('new' when bootstrapping a new cluster or 'existing' when adding new members to an existing cluster).Initial cluster state is based on the availability of a populated data/WAL directory.")
```

Comment:
Suggest to remove "`Initial cluster state is based on the availability of a populated data/WAL directory.`". Note that there shouldn't have local data no matter bootstrapping a new cluster or adding new members into an existing cluster. 

After successfully bootstrapping the new cluster or adding any new members, then it doesn't matter what's the value for the flag.

Reply:
How about we also add `After successful initialization (bootstrapping or adding), flag is ignored on restarts`

Reply:
Agreed.

Reply:
done!

Reply:
@ahrtr and @lavacat, squashed the commits and updated the desc based on the review inputs